spec/opentok/archives_spec.rb

```diff
@@ -57,6 +57,17 @@
     expect(archive.output_mode).to eq :individual
   end
 
+  it "should create custom layout archives", :vcr => { :erb => { :version => OpenTok::VERSION + "-Ruby-Version-#{RUBY_VERSION}-p#{RUBY_PATCHLEVEL}" } } do
+    custom_layout = {
+      :type => "custom",
+      :stylesheet => "stream:last-child{display: block;margin: 0;top: 0;left: 0;width: 1px;height: 1px;}stream:first-child{display: block;margin: 0;top: 0;left: 0;width: 100%;height: 100%;}"
+    }
+    archive = archives.create session_id, :layout => custom_layout
+    expect(archive).to be_an_instance_of OpenTok::Archive
+    expect(archive.session_id).to eq session_id
+    expect(archive.layout).to eq custom_layout
```

Comment:
There is no `layout` property of an Archive object. And I don't think there should be. (We don't expose it in the other server SDKs.) Please remove this line.